orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/Front50Service.groovy

```diff
@@ -52,6 +52,9 @@ interface Front50Service {
   @GET("/pipelines/{applicationName}")
   List<Map<String, Object>> getPipelines(@Path("applicationName") String applicationName, @Query("refresh") boolean refresh)
 
+  @GET("/pipelines/{pipelineId}/history")
+  List<Map<String, Object>> getPipelineHistory(@Path("pipelineId") String pipelineId, @Query("limit") int limit)
```

Comment:
Sadly we don't currently have an api to fetch by pipelineId ... this is the closest thing to it.

I've optimized `front50` such that calling`getPipelineHistory` with `limit = 1` will short-circuit to a simple load without worrying about fetching versions.

Reply:
I'd love to revisit the Front50 endpoints around pipeline configs and maybe make them so we can send in an id or name and avoid things like this and [like this](https://github.com/spinnaker/gate/blob/master/gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/ApplicationService.groovy#L158)